cmd/emulator/start/start.go

```diff
@@ -73,6 +73,7 @@ type Config struct {
 	ChainID                   string        `default:"emulator" flag:"chain-id" info:"chain to emulate for address generation. Valid values are: 'emulator', 'testnet', 'mainnet'"`
 	RedisURL                  string        `default:"" flag:"redis-url" info:"redis-server URL for persisting redis storage backend ( redis://[[username:]password@]host[:port][/database] ) "`
 	SqliteURL                 string        `default:"" flag:"sqlite-url" info:"sqlite db URL for persisting sqlite storage backend "`
+	CoverageReportingEnabled  bool          `default:"true" flag:"coverage-reporting" info:"enable Cadence code coverage reporting"`
```

Comment:
probably better to not be enabled by default?

Reply:
why resolved?

Reply:
sorry I misclicked on this probably

Reply:
Good point, I will change the default behavior :+1:

Reply:
Updated the default value to `false`